go/chat/flipmanager_test.go

```diff
@@ -75,7 +82,53 @@ func TestFlipManagerStartFlip(t *testing.T) {
 				consumeNewMsgRemote(t, listener2, chat1.MessageType_SYSTEM)
 			}
 
+			// ensure we only include the channel participants in the flip for team
+			// conversations.
+			expectedDevConvs := 0
```

Comment:
Can we move this to a different test that doesn't run so many times? It seems independent of members type.

Reply:
yeah good idea